lib/medium-admin.php

```diff
@@ -136,10 +136,16 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
+
+    if (!in_array(get_post_type($post_id), $allowed_post_types)) {
+      return;
+    }
+
     $medium_post = Medium_Post::get_by_wp_id($post_id);
 
     // If this post has already been sent to Medium, no need to do anything.
-    if ($medium_post->id) return;
+    if (!empty($medium_post->id)) return;
```

Comment:
@majelbstoat I found the problem, it's because the `$post` being passed inside the `save_post` action is a post revision, not the original parent post. Making a fix now...